### ISSUE TITLE
I can revert the code easily ;)

### ISSUE THREAD

**Issue by tbvinh** [pH-7/Obfuscator-Class#26]
#Edit the obfuscate file:
1. Find the line: "\$\___();\$\__________(\$\______(\$\__(\$_))); \$\________=\$\____();"
2. Delete all to end of file
3. Add some lines to see the original source code:
\--------------------------------------
\$source = base64_decode(gzuncompress(base64_decode($_)));
\$idx = strpos($source, "eval(\\\$___)");
\$orgSource = substr($source, $idx + 24);
echo "\<br/>ORIGIN SOURCE\<br/>\<pre> $orgSource \</pre> \<br/>";
\---------------------------------------

Please see the php code at my gist:
https://gist.github.com/tbvinh/d3a833321b4ba7e292253aa5a76ef52e

**Comment by MasterB0t** [comment]
Yes, within 5 minutes I figure it out too .. its just a "stupid base64 encode script"

**Comment by pH-7** [comment]
Thanks for your suggestions! I appreciate this. Would you like to add more level of complexity and obfuscation? I'm more than happy to review any of your PRs you want to add here 🥳

**Comment by MasterB0t** [comment]
Try this one https://github.com/pk-fr/yakpro-po
its a really good one, it rename functions, classes, variable in to random characters it also add a lot of goto statement to make it even more hard to read.

**Comment by 2braincells2go** [comment]
@MasterB0t How is Yak better and harder to decode? I assume you can tell us the difference and how it is better.

**Comment by MasterB0t** [comment]
@2braincells2go  sure, 
This software use base64_encode and gzcompress to encode the php code you can use base64_decode and gzuncompress and you will get the original code. Those function are not to encode source code or obfuscate it, made it hard to understand. 

Yakpro in other hard transform code, it rename variables, functions, class name, method into random characters so if you have a function something like this 
function login($username,$password) {  }
it turn it into 
function dd30d0uj09r03uru30r03ur($odj0303ui0d3ur,$fj03j0j30jf30) {} 
but also use the the goto statement so the code is not in order from line 1 to bottom ... you can read more about goto [here](https://www.php.net/manual/en/control-structures.goto.php)

**Comment by 2braincells2go** [comment]
@MasterB0t Thank you for explaining! Makes perfect sense now:)

**Comment by parallels999** [comment]
@tbvinh it works, it is just base64 encoding, it doesn't ofuscate anything
![image](https://user-images.githubusercontent.com/109294935/199110998-5b76717e-2a8a-49be-8ab7-0f19f8e31aeb.png)

**Comment by parallels999** [comment]
https://github.com/pH-7/Obfuscator-Class/blob/bf7ad9cefa63dd40ec1109cc2ee240557adb3486/README.md?plain=1#L5
![image](https://user-images.githubusercontent.com/109294935/199112193-020b81fb-80b9-4be6-bdf1-590dfebcdc51.png)
